### ISSUE TITLE
How can I measure Novel-View Synthesis and Self-Reenactment?

### ISSUE THREAD

**Issue by HoYoung-City** [ShenhanQian/GaussianAvatars#24]
After finishing training, I measured Novel-View Synthesis and Self-Reenactment on my own with `metrics.py`, but I'm not sure if I did it correctly.

To render for Novel-View Synthesis, attaching only `--skip_train` and `--skip_val` as below, is that correct?
Or is there any other operation that needs to be done?

```
SUBJECT=306

python render.py \
-m output/UNION10EMOEXP_${SUBJECT}_eval_600k \
--skip_train --skip_test
```

Additionally, to perform Self-Reenactment, can it be done as below?
```
SUBJECT=306
TGT_SUBJECT=306

python render.py \
-t data/UNION10_${TGT_SUBJECT}_EMO1234EXP234589_v16_DS2-0.5x_lmkSTAR_teethV3_SMOOTH_offsetS_whiteBg_maskBelowLine \
-m output/UNION10EMOEXP_${SUBJECT}_eval_600k
```

It might be that I don’t fully understand Novel-View Synthesis and Self-Reenactment, but I’m not clear on the difference between the two.
Thank you for your research.
I am currently conducting research using your study as a baseline.
I would like to measure the metrics correctly if you could inform me of the proper method.

**Comment by ShenhanQian** [comment]
Hi, we conduct evaluations during the training process. 

You could find the metrics printed in the command line and logged in Tensorboard. The validation set is for novel-view synthesis; the test set is for self-reenactment.

**Comment by HoYoung-City** [comment]
Thank you for your kind response!
Once again, I appreciate your research.
